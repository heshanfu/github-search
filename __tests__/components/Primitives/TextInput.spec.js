import renderer from 'react-test-renderer';
import 'jest-styled-components';
import React from 'react';

import TextInput from '../../../app/components/Primitives/TextInput';

describe('<TextInput />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TextInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
