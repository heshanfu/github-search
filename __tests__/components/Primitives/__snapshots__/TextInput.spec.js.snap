// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextInput /> renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  style={
    Array [
      Object {
        "alignSelf": "stretch",
        "borderColor": "#222",
        "borderWidth": 1,
        "height": 40,
        "paddingLeft": 5,
      },
      undefined,
    ]
  }
/>
`;
